### PR TITLE
[CMS-867] Use right pantheon api url.

### PR DESCRIPTION
**Pull request by kporras07**

### CODE REVIEW

File: src/SecretsApi/SecretsApi.php [hunk 1]
```diff
@@ -22,11 +22,24 @@ class SecretsApi
     private function getBaseURI()
     {
         $config = $this->request()->getConfig();
+
+        $protocol = $config->get('papi_protocol') ?? $config->get('protocol');
+        $port = $config->get('papi_port') ?? $config->get('port');
+        // @todo Change to pantheonapi.svc.pantheon.io once alias is set.
+        $host = 'pantheonapi.production.general-01.us-central1.internal.k8s.pantheon.io';
+        if (!empty($config->get('papi_host'))) {
+            $host = $config->get('papi_host');
```
Comment (greg-1-anderson) [inline src/SecretsApi/SecretsApi.php:31]: We could reduce the cyclometric complexity of this function if we start by assigning 'papi_host' to $host, and then check $terminusHost iff $host is empty, and then finally plug in the default value if $host is still empty. Same number of `if`s, but we remove an else and one level of nesting.

**Review by greg-1-anderson — CHANGES_REQUESTED** [review]
Code looks good, just refactor a bit.

File: src/SecretsApi/SecretsApi.php (hunk 1 above)
Comment (kporras07) [reply src/SecretsApi/SecretsApi.php:31]: Done. Please take another look

File: src/SecretsApi/SecretsApi.php [hunk 2]
```diff
@@ -22,11 +22,24 @@ class SecretsApi
     private function getBaseURI()
     {
         $config = $this->request()->getConfig();
+
+        $protocol = $config->get('papi_protocol') ?? $config->get('protocol');
+        $port = $config->get('papi_port') ?? $config->get('port');
+        $host = $config->get('papi_host');
+        if (!$host && strpos($config->get('host'), 'hermes.sandbox-') !== false) {
+            $host = 'pantheonapi.sandbox-eco.sbx01.pantheon.io';
```
Comment (greg-1-anderson) [inline src/SecretsApi/SecretsApi.php:30]: This shouldn't be hardcoded to `sandbox-eco`; take it from `hermes.sandbox-NAME` in the config `host` variable.

**Review by greg-1-anderson — CHANGES_REQUESTED** [review]
Select the sandbox that matches the TERMNUS_HOST

File: src/SecretsApi/SecretsApi.php (hunk 2 above)
Comment (kporras07) [reply src/SecretsApi/SecretsApi.php]: That makes sense. Fixed

**Review by greg-1-anderson — APPROVED** [review]

**Pull request merged**